flask/app/routes.py

```diff
@@ -517,3 +518,45 @@ def delete_tissue(id: int):
         return "Tissue has dataset(s), cannot delete", 422
     else:
         return "Not Found", 404
+
+
+@app.route("/api/families", methods=["POST"])
+@login_required
+def families():
```

Comment:
```suggestion
def create_family():
```
This should be in `families.py`